### ISSUE TITLE
Spoof different android versions

### ISSUE THREAD

**Issue by swiftgeek** [Lekensteyn/apk-downloader#18]
Some apps provide different apks for different android versions
http://developer.android.com/google/play/publishing/multiple-apks.html

**Comment by Lekensteyn** [comment]
Do you have an example of such an app?

**Comment by swiftgeek** [comment]
https://play.google.com/store/apps/details?id=com.ebay.mobile&hl=en
(not a reason for feature request but i had to find some neutral one)

**Comment by landry314** [comment]
yes, and you can't download an apk of a text messaging app on a tablet, for example... this is another reason to spoof.

**Comment by mfonville** [comment]
It would also be great to be able to switch between (spoofed) 'profiles' to be able to download multiple APK versions.
